src/types.jl

```diff
@@ -98,6 +98,16 @@ end
 ###
 ### Term interface
 ###
+
+"""
+  symtype(x)
```

Comment:
```suggestion
    symtype(x)
```
A four-space indent is needed so that it is printed as Julia code.
See https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation